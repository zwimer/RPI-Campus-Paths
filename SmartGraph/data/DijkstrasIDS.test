r
51
43
r
43
51
q