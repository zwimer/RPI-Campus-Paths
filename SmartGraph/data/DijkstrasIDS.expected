First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from 2144 Burdett Avenue to Sharp Hall:
	Walk NorthWest to (Intersection 113)
	Walk South to (Intersection 109)
	Walk South to (Intersection 124)
	Walk SouthWest to (Crockett Hall)
	Walk SouthWest to (Intersection 123)
	Walk South to (Davison Hall)
	Walk SouthWest to (Sharp Hall)
Total distance: 749.925 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from Sharp Hall to 2144 Burdett Avenue:
	Walk NorthEast to (Davison Hall)
	Walk North to (Intersection 123)
	Walk NorthEast to (Crockett Hall)
	Walk NorthEast to (Intersection 124)
	Walk North to (Intersection 109)
	Walk North to (Intersection 113)
	Walk SouthEast to (2144 Burdett Avenue)
Total distance: 749.925 pixel units.
